config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   root to: 'posts#index'
   resources :posts
+  post '/posts/new', to: 'posts#new'
+
   resources :users
+  post '/users/:id/edit', to: 'users#edit'
```

Comment:
Now, why `resources :users` doesn't have `post '/users/:id/edit'` already covered is interesting to me.  Same with `:posts`.  But with the switch from links to buttons, these routes were being requested (via error, that is to say haha)